port/unix/omrsysinfo.c

```diff
@@ -4642,7 +4642,7 @@ omrsysinfo_get_CPU_load(struct OMRPortLibrary *portLibrary, double *cpuLoad)
 	if (oldestCPUTime->timestamp == 0) {
 		*oldestCPUTime = currentCPUTime;
 		*latestCPUTime = currentCPUTime;
-		return OMRPORT_ERROR_OPFAILED;
+		return OMRPORT_ERROR_INVALID;
```

Comment:
Its safer to add a new error code for this, perhaps `OMRPORT_ERROR_INSUFFICIENT_DATA`